modules/simple-payments/simple-payments.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+class Jetpack_Simple_Payments {
+	// These have to be under 20 chars because that is CPT limit.
+	static $post_type_order = 'jp_pay_order';
+	static $post_type_product = 'jp_pay_product';
+
+	static $shortcode = 'simple-payment';
```

Comment:
Not weird for me. It's a product name (like videopress), not a real English word. It would make sense in case of `[paypal-button id...]` but `[simple-payment]` is strange as the product is called `simple-payments`. 

/cc @jhnstn